static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -529,8 +546,25 @@ const StyledSearchBar = styled(SearchBar)`
   `}
 `;
 
+const OnDemandMetricInfoAlert = styled(Alert)`
+  border-radius: ${space(0.5)} ${space(0.5)} 0 0;
+  border: none;
+  border-bottom: 1px solid ${p => p.theme.blue400};
+  margin-bottom: 0;
+
+  & > span {
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const OnDemandMetricInfoIcon = styled(IconInfo)`
+  color: ${p => p.theme.blue400};
```

Comment:
is this color ok in dark mode?

Reply:
Looks good!
![image](https://github.com/getsentry/sentry/assets/86684834/78282505-8adf-465a-b964-a46a8da5aa75)